package.json

```diff
@@ -141,13 +198,97 @@
+                    "group": "navigation@1",
+                    "when": "view == lf-lang-library"
+                },
+                {
+                    "command": "linguafranca.collapseAll",
+                    "group": "navigation@2",
+                    "when": "view == lf-lang-local"
+                },
+                {
+                    "command": "linguafranca.collapseAllLibrary",
+                    "group": "navigation@2",
+                    "when": "view == lf-lang-library"
+                }
+
+            ],
+            "view/item/context" : [
+                {
+                    "command": "linguafranca.goToFile",
+                    "group": "inline",
+                    "when": "view == lf-lang-local && viewItem != root"
+                },
+                {
+                    "command": "linguafranca.goToLibraryFile",
+                    "group": "inline",
+                    "when": "view == lf-lang-library && viewItem != root"
+                },
+                {
+                    "command": "linguafranca.openInSplitView",
+                    "group": "inline",
+                    "when": "view == lf-lang-local && viewItem == file"
+                },
+                {
+                    "command": "linguafranca.openLibraryInSplitView",
+                    "group": "inline",
+                    "when": "view == lf-lang-library && viewItem == file"
+                },
+                {
+                    "command": "linguafranca.openInSplitView",
+                    "when": "view == lf-lang-local && viewItem == reactor"
+                },
+                {
+                    "command": "linguafranca.openLibraryInSplitView",
+                    "when": "view == lf-lang-library && viewItem == reactor"
+                },
+                {
+                    "command": "linguafranca.importReactor",
+                    "group": "inline",
+                    "when": "view == lf-lang-local && viewItem == reactor"
+                },
+                {
+                    "command": "linguafranca.importLibraryReactor",
+                    "group": "inline",
+                    "when": "view == lf-lang-library && viewItem == reactor"
+                }
+            ]
+        },
+        "viewsContainers": {
+            "activitybar": [
+                {
+                    "id": "lf-lang",
+                    "title": "Lingua Franca Package Explorer",
+                    "icon": "images/logo/lf-logo-dark.svg"
+                }
+            ]
+        },
+        "views": {
+            "lf-lang": [
+                {
+                    "id": "lf-lang-local",
+                    "name": "Local Libraries"
+                },
+                {
+                    "id": "lf-lang-library",
+                    "name": "Lingo Libraries"
+                }
+              ]
+        },
+        "commandPalette": [
```

Comment:
Why did the commandPalette move out of the configuraton?

Reply:
I think it was a merge conflict mistake. Where should I place it in the structure?